### ISSUE TITLE
How do I determine if a signal is declared?

### ISSUE THREAD

**Issue by pajama** [ssannandeji/Zenject-2019#629]
I would like to be able to optionally subscribe to a signal, but only if has been declared in my current context/parent contexts. Is that possible?

**Comment by svermeulen** [comment]
Yeah there wasn't really a way to do this.  I've fixed this in https://github.com/svermeulen/Extenject by adding a `SignalBus.IsSignalDeclared` method
